### PR TITLE
⬆️ react-scripts to 3.2.0

### PR DESCRIPTION
**Pull request by dmihalcik-virtru**
This will up he transitive dep on `eslint-utils` to 1.4.3, fixing our vulnerability alert

### CODE REVIEW

File: license-update [hunk 1]
```diff
@@ -3,6 +3,10 @@
 license=$(sed -e 's|^#*|// |' <LICENSE)
 
 function _process {
+  if [ ! -f "$FILE" ]
+  then
+    return 0
+  fi
```
Comment (ivanovSPvirtru) [inline license-update]: Package lock and json fix are clear, but what is those lines and user select scene deleted

Comment (jrschumacher) [reply license-update]: @ivanovSPvirtru These lines handle when a file is removed. The script will pick up a change to the file, but the file no longer exists and can't add a license header.

Comment (jrschumacher) [reply license-update]: Regarding the `UserScene` file I don't believe they were used. Checking.

Comment (jrschumacher) [reply license-update]: Confirmed: https://github.com/virtru/protect-and-track/search?q=userselect&unscoped_q=userselect

**Review by jrschumacher — APPROVED** [review]

File: license-update (hunk 1 above)
Comment (ivanovSPvirtru) [reply license-update]: Gotcha!

**Pull request merged**